packages/peregrine/lib/talons/CartPage/PriceAdjustments/ShippingMethods/useShippingRadios.js

```diff
@@ -0,0 +1,60 @@
+import { useCallback, useEffect } from 'react';
+import { useFieldApi } from 'informed';
+import { useMutation } from '@apollo/react-hooks';
+
+import { useCartContext } from '../../../../context/cart';
+
+export const useShippingRadios = props => {
+    const {
+        selectedShippingMethod,
+        setShippingMethodMutation,
+        shippingMethods
+    } = props;
+    const shippingMethodFieldApi = useFieldApi('method');
+    const [setShippingMethod] = useMutation(setShippingMethodMutation);
+    const [{ cartId }] = useCartContext();
+
+    const formattedShippingMethods = shippingMethods.map(shippingMethod => ({
+        ...shippingMethod,
+        serializedValue: `${shippingMethod.carrier_code}|${
+            shippingMethod.method_code
+        }`
```

Comment:
I'm a little wary of the format of this value having to match usage in `useShippingMethods`, but since it's only two places at the moment I think it falls short of an official change request 😄

Reply:
Yeah. I really would have preferred just using objects as the value, but felt this was okay since the serialization is contained to the talon.